src/Gateway/Signup.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace VendoSdk\Gateway;
+
+use VendoSdk\Exception;
+use VendoSdk\Gateway\Request\Details\CreditCard;
+use VendoSdk\Gateway\Request\Details\SubscriptionSchedule;
+
+/**
+ * Class Signup
+ * @package VendoSdk\Gateway
+ *
+ */
+class Signup extends PaymentBase implements \JsonSerializable
```

Comment:
Again, if we're using the same /payment endpoint, signups should not its own class. A signup is just a payment with a subscription schedule entity.

Reply:
ok

Reply:
removed Signup class